### PR TITLE
[feature](ui) web profile download use plain text

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

download the plain text of the profile, reduce the overhead of  string replace , converting JSON format.
```java
    @RequestMapping(path = "/query_profile/{" + ID + "}", method = RequestMethod.GET)
    public Object profile(@PathVariable(value = ID) String id) {
        String profile = ProfileManager.getInstance().getProfile(id);
        if (profile == null) {
            return ResponseEntityBuilder.okWithCommonError("ID " + id + " does not exist");
        }
        profile = profile.replaceAll("\n", "</br>");
        profile = profile.replaceAll(" ", "&nbsp;&nbsp;");
        return ResponseEntityBuilder.ok(profile);
    }
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/QueryProfileController.java [hunk 1]
```diff
@@ -59,6 +59,15 @@ public Object profile(@PathVariable(value = ID) String id) {
         return ResponseEntityBuilder.ok(profile);
     }
 
+    @RequestMapping(path = "/query_profile/text/{" + ID + "}", method = RequestMethod.GET)
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/QueryProfileController.java:62]: when this API will be used?

Comment (Mryange) [reply fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/QueryProfileController.java:62]: web use download

Comment (HappenLee) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/QueryProfileController.java:62]: we better improve the performance of `replaceAll("\n", "</br>");`

**Pull request closed without merging**